containers/app/Dockerfile

```diff
@@ -32,21 +32,32 @@ FROM python:3.12-slim as runtime
 
 WORKDIR /app
 
-ENV RUN_AS_DEVIN=false
-ENV USE_HOST_NETWORK=false
+ENV RUN_AS_DEVIN=true
+ENV SANDBOX_USER_ID=1000
+ENV USE_HOST_NETWORK=true
```

Comment:
I still think we need to leave host network off until it's out of beta for Docker Desktop users. Is this a critical part of this PR?

Reply:
reverted!